db/db.py

```diff
@@ -0,0 +1,50 @@
+import pymysql
+import logging
+
+
```

Comment:
Remove extra new line (example from elsewhere in the code: https://github.com/chaosbot/Chaos/blob/9dabaf3349f64c9b7a4b9d32e3a1ccb7afa8122e/cron/poll_issue_close_stale.py#L7)